.github/workflows/tag-and-build.yml

```diff
@@ -0,0 +1,76 @@
+# This workflow will build the CodeFlare Operator image and push it to the project-codeflare image registry
+
+name: Operator Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Tag to be used for operator image'
+        required: true
+        default: '0.0.0-dev'
+      replaces:
+        description: 'The previous semantic version that this tag replaces.'
+        required: true
+        default: '0.0.0-dev'
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Activate cache
+      uses: actions/cache@v2
+      with:
+        path: /cache
+        key: ${{ runner.os }}-cache-${{ hashFiles('**/go.sum', '.pre-commit-config.yaml') }}
+
+    - name: Create tag
+      uses: actions/github-script@v5
+      with:
+        script: |
+          github.rest.git.createRef({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            ref: 'refs/tags/${{ github.event.inputs.version }}',
+            sha: context.sha
+          })
+
+    - name: Install operator-sdk
+      run: |
+        export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+        export OS=$(uname | awk '{print tolower($0)}')
+        export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.27.0
+        curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+        chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
```

Comment:
Is there a reason not using `make install-operator-sdk` ?

Reply:
I was using an existing git action. https://github.com/project-codeflare/codeflare-operator/blob/main/.github/workflows/operator-image.yml I can see if I can do this the way that you suggested